static/bindata.go

```diff
@@ -120,12 +120,12 @@ func scriptsWeb3Js() (*asset, error) {
 		return nil, err
 	}
 
-	info := bindataFileInfo{name: "scripts/web3.js", size: 497689, mode: os.FileMode(420), modTime: time.Unix(1507576052, 0)}
+	info := bindataFileInfo{name: "scripts/web3.js", size: 497689, mode: os.FileMode(420), modTime: time.Unix(1507747781, 0)}
```

Comment:
Expecting these times to be proper defined.

Reply:
Sorry?

Reply:
The unix times are changed here and it's not directly obvious to me where this value is come from.

Reply:
`make generate`

Reply:
Ah, top, thanks. 👍